substrate/client/chain-spec/src/chain_spec.rs

```diff
@@ -1010,6 +1035,38 @@ mod tests {
 	}
 
 	#[docify::export]
```

Comment:
now sure where it is used, but happy to see this :)

Reply:
https://paritytech.github.io/polkadot-sdk/master/sc_chain_spec/index.html#building-a-chain-specification

Reply:
Hmm, but that uses `build_chain_spec_with_patch_works` and not `generate_chain_spec_with_named_preset_works`, but both are exported via docify, did you intend to replace this maybe?

Reply:
just blindly copied, fixed. `build_chain_spec_with_patch_works` is still used.